### ISSUE TITLE
Add feature that supports importing existing CubeFS volume into k8s cluster and create a corresponding PersistentVolume

### ISSUE THREAD

**Issue by Huweicai** [cubefs/cubefs-csi#70]

**Comment by GongWilliam** [comment]
Does any situation requires this feature?

**Comment by Huweicai** [comment]
For example, if I have two Kubernetes cluster, they are using a same background CubeFS cluster, but what if I want to share a volume between these two clusters?
It's hard to share volumes now, we have to create and fill up a custom PV and bind it to PVC manually.

**Comment by dantegarden** [comment]
Honestly，I have same problem.

**Comment by Weyn** [comment]
Hi @Huweicai did you manage to get that sorted or add some way to use existing volume? In our case we do share volumes between different services so we would like to be able to target existing volume rather than create dynamic ones.

**Comment by Huweicai** [comment]
@Weyn Nope, i am not working on it now, but there's a way to achieve it manually, you can copy all the `volumeAttributes` from one and create a PV in another cluster with the same `volumeAttributes`.
